test/test_prototype_transforms.py

```diff
@@ -2350,7 +2344,7 @@ def test_detection_preset(image_type, data_augmentation, to_tensor, sanitize):
         # Note that the values below are probably specific to the random seed
         # set above (which is fine).
         (True, "ssd"): 4,
-        (True, "ssdlite"): 3,
```

Comment:
I guess this coming from the bug fix below, but we probably need to double check again. At least 4 < 5, so we are at least removing at least 1 box.

Reply:
I've checked this again manually. The transform is doing what it should be doing. The fact that we need to change here comes from the fixed behavior in https://github.com/pytorch/vision/pull/7268#discussion_r1108608227. I guess we just lucked out on ssd.